src/lib/provas/2019/17.json

```
@@ -0,0 +1,18 @@
+{
+    "ano": "2019",
+    "questao": "17",
+    "area": "Matemática",
+    "enunciado": "Uma pessoa deseja fazer uma compra na Internet e, para isso, precisa se cadastrar \nem um site. A senha de cadastro deve ser formada por exatamente 9 caracteres, e somente os \ncaracteres $, @ e # podem ser usados. Quantas senhas diferentes, contendo pelo menos uma \nocorrência de cada caractere, existem?",
```

Acredito que os "\n" podem ser removidos, e deixar por conta do proprio site a quebra de linha.

```suggestion
    "enunciado": "Uma pessoa deseja fazer uma compra na Internet e, para isso, precisa se cadastrar em um site. A senha de cadastro deve ser formada por exatamente 9 caracteres, e somente os caracteres $, @ e # podem ser usados. Quantas senhas diferentes, contendo pelo menos uma ocorrência de cada caractere, existem?
```